ee/tabby-db/src/user_completions.rs

```diff
@@ -61,6 +68,71 @@ impl DbConn {
         Ok(())
     }
 
+    // FIXME(boxbeam): index `created_at` in user_completions table.
+    pub async fn compute_daily_stats_in_past_year(
+        &self,
+        users: Vec<i32>,
+    ) -> Result<Vec<UserCompletionDailyStatsDAO>> {
+        let users = users
+            .iter()
+            .map(|u| u.to_string())
+            .collect::<Vec<_>>()
+            .join(",");
+        Ok(sqlx::query_as(
```

Comment:
Any reason not to make this type use `DateTimeUtc` and `i64`, and use `query_as!` instead?